lib/ui/profile/presenter/profile_tweets.dart

```diff
@@ -0,0 +1,10 @@
+import 'package:flutter_dasher/common/model/tweet.dart';
+import 'package:flutter_dasher/domain/interactor/profile/profile_tweets_interactor.dart';
+import 'package:riverpod_annotation/riverpod_annotation.dart';
+
+part 'profile_tweets.g.dart';
+
+@riverpod
+Future<List<Tweet>> profileTweets(ProfileTweetsRef ref) {
```

Comment:
This is not possible. Riverpod will always add the `Provider` suffix.
We can change the suffix, but it will change it for all providers ([issue](https://github.com/rrousselGit/riverpod/pull/1898)).

We should definitively discuss the naming (should it be `profileTweets`, `profileTweetsPresenter`, `profileTweetsNotifier`).